index.html

```diff
@@ -64,12 +64,14 @@ <h3>Controls</h3>
                 </tbody>
             </table>
             
-            <div id='toggle-logos' class='on' onclick='toggleLogos()'>
-                <p class='main'><span id='logos-off'>Enable</span><span id='logos-on'>Disable</span> Custom Team Logos?</p>
-                <p class='sub'>(Reloads Page)</p>
+            <div id='toggle-logos'>
```

Comment:
So close!

Reply:
My bad.  Must have accidentally switched it back to single quotes when restructuring the elements in order to fix the Edge CSS glitch.  Should be fixed now.